game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    user_wins = is_word_guessed(snowman_word, correct_letter_guess_statuses)
+
+    while (
+        (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES)
+        and (not user_wins)
+    ):
+        user_input = get_letter_from_user(correct_letter_guess_statuses, 
+                                        wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
```

Comment:
Note that the `if user_guess in correct_letter_guess_statuses` is shorthand ([in and not in](https://docs.python.org/3/reference/expressions.html#in)) for: 
```py
for letter in snowman_word: 
    if user_guess == letter: 
        return True
    return False
```

You can use this syntax to perform the above logic over a multiple kinds of iterables/sequences in Python. Later in the curriculum we will learn that they don't all have the same performance (Big O).